components/ContentContainer/index.tsx

```diff
@@ -0,0 +1,19 @@
+'use client'
+
```

Comment:
Created a basic ContentContainer component that will wrap content in components for the shared content background and possibly new functionality.